src/framework/graphics/skinned-mesh.zig

```diff
@@ -220,17 +220,23 @@ pub const SkinnedMesh = struct {
         if (self.joint_locations_dirty)
             self.applySkeletonTransforms();
 
-        self.mesh.material.state.params.joints = &self.joint_locations;
-        graphics.drawWithMaterial(&self.mesh.bindings, &self.mesh.material, cam_matrices, model_matrix);
+        // TODO check this because was from mesh only had one material
```

Comment:
this I have no idea, need guidance

Reply:
The params are the uniforms that get passed to the shader - since each material is a new draw call each material would also need to have their joint state updated as well otherwise only the first material would be animated.